packages/file-viewer/src/components/file-viewer.tsx

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable import/no-duplicates */
+import * as React from 'react';
+import { useState } from 'react';
+import styled from '@emotion/styled';
+import { colors, gridSize, themed, math } from '@atlaskit/theme';
+import CodeIcon from '@atlaskit/icon/glyph/code';
+
+import Prism from 'prismjs';
+import 'prismjs/components/prism-jsx';
+import 'prismjs/themes/prism-tomorrow.css';
+
+// TODO Can't copy ErrorBoundary (parent of Component to handle errors gracefully) from Atlaskit
+// because componentDidCatch has no hook equivalent yet. Implement once support is added.
```

Comment:
Just making sure I understand - this means that if the code in the example is broken, the whole site will crash for now?

I think we should convert this back to a class component so we can handle this.